### DISCUSSION TITLE
Remove this line

### DISCUSSION THREAD

**Discussion by khill-fbmc** [sapegin/mrm#318] [Ideas]
https://github.com/sapegin/mrm/blob/4e1d4c81f88c8dfcfb94c44d136c808fac591499/packages/mrm-task-eslint/index.js#L126

[As per the documentation, this is obsolete and should be removed](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21)

**Comment by kevinkhill** [comment]
It looks like someone already mentioned this here #169 too
